imagecraft/services/pack.py

```diff
@@ -36,38 +35,23 @@ def __init__(
         app: AppMetadata,
         services: "ImagecraftServiceFactory",
         *,
-        project: models.Project,
+        project: Project,
         build_plan: list[BuildInfo],
     ) -> None:
         super().__init__(app, services, project=project)
+
         self._build_plan = build_plan
 
     @override
-    def pack(self, prime_dir: pathlib.Path, dest: pathlib.Path) -> list[pathlib.Path]:
+    def pack(self, prime_dir: pathlib.Path, dest: pathlib.Path) -> list[pathlib.Path]:  # noqa: ARG002
         """Pack the image.
 
         :param prime_dir: Directory path to the prime directory.
         :param dest: Directory into which to write the package(s).
         :returns: A list of paths to created packages.
         """
-        gadget_path = f"{prime_dir}/gadget/"
-        rootfs_path = f"{prime_dir}/rootfs/"
-        workdir_path = f"{prime_dir}/workdir/"
-
-        # Create per-platform output directories
-        platform_output = pathlib.Path(
-            dest,
-            self._build_plan[0].platform if self._build_plan[0].platform else "",
-        )
-        platform_output.mkdir(parents=True, exist_ok=True)
-
-        ubuntu_image_pack(rootfs_path, gadget_path, str(dest), workdir_path)
-
-        img_paths = list_image_paths(workdir_path)
-
-        shutil.rmtree(workdir_path)
 
-        return [pathlib.Path(dest) / img for img in img_paths]
+        return []
```

Comment:
task-wise, what's the breakdown here? Is this coming in a future task?

Reply:
Correct. I still wanted to keep the ImagecraftPackService around for now though since it may be a central part in imagecraft. I am not sure if this is noise or not though. What do you think?

Reply:
I think this is fine